features/step_definitions/scheduler.rb

```diff
@@ -7,11 +7,11 @@
   }
 end
 
-Given /^the #{QUOTED} scheduler CR is restored after scenario$/ do |name|
+Given /^the #{QUOTED} #{QUOTED} scheduler CR is restored after scenario$/ do |name, crd|
```

Comment:
How about `Given /^the CR #{QUOTED} named #{QUOTED} is restored after scenario$/ do |type, name|`

Reply:
should be fine i guess, i will update it accordingly

Reply:
Addressed